_events/2023.md

```diff
@@ -13,12 +13,14 @@ year: 2023
 
 ## Event Information
 
+It’s finally here!  BSides Roanoke is live and we are ready bring you the best and most exciting security talks in the Roanoke Valley and beyond.  We are actively working to bring you the talks that you want and that are relevant to you and your organization.   More content is being added daily so please be patient as we continue to work and mold this event to be the best one yet.  
```

Comment:
It's helpful to put one sentence per line for easier editing.

Given that this is a weekend event, should we adjust the message to speak to the individual vs to speaking to an organization?

```suggestion
It’s finally here!  BSides Roanoke is live and we are ready bring you the best and most exciting security talks in the Roanoke Valley and beyond.  We are actively working to bring you the talks that you want and are relevant to you.   More content is being added daily so please be patient as we continue to work and mold this event to be the best one yet.  
```